packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -137,10 +140,13 @@ class CallContext {
 
     // As the state is transitioning to a new state, trigger appropriate callback events.
     const oldPage = this.state.page;
+    /* @conditional-compile-remove(unsupported-browser) */
+    const environmentInfo = this.state.features?.unsupportedEnvironment ? this.state.environmentInfo : undefined;
```

Comment:
this check is unnecessary as the getCallCompositePage does this check (either keep here and remove from getCallCompositePage or remove from here)